core/src/main/scala/com/rallyhealth/vapors/factfilter/extras/ExtractInstant.scala

```diff
@@ -0,0 +1,24 @@
+package com.rallyhealth.vapors.factfilter.extras
+
+import java.time.{Instant, LocalDate, LocalDateTime, ZoneId, ZonedDateTime}
+
+import com.rallyhealth.vapors.factfilter.data.ExtractValue
+
+trait ExtractInstant[-T] extends ExtractValue[T, Instant]
```

Comment:
This should either only live in the test code as an example (which can be copied into Kerna) or be moved out into a separate subproject.

Reply:
This is used by the algebra now for And / Or expressions.